auxx/src/Command/Tx.hs

```diff
@@ -199,7 +202,10 @@ send sendActions idx outputs = do
         let getSigner = fromMaybe (error "Couldn't get SafeSigner") . flip HM.lookup addrSig
         -- BE CAREFUL: We create remain address using our pk, wallet doesn't show such addresses
         (txAux,_) <- lift $ prepareMTx getSigner mempty def (NE.fromList allAddresses) (map TxOutAux outputs) curPk
-        txAux <$ (ExceptT $ try $ submitTxRaw (immediateConcurrentConversations sendActions ccPeers) txAux)
+        let txAux' =
+                if not fakeInputs then txAux
+                else txAux { taTx = taTx txAux & txInputs %~ fmap (\x -> x { txInIndex = 100500 }) }
```

Comment:
Btw, what will `x { txInIndex = 100500 }` do if `x` is `TxInUnknown`?

Reply:
It will crash. I was unaware that there's TxInUnknown.